### ISSUE TITLE
ortoni-report is not generating for Mac

### ISSUE THREAD

**Issue by nirmalprojects1988** [ortoniKC/ortoni-report#13]
Steps to reproduce this issue

1. Ran the command npm i ortoni-report.
2. Added the code in playwright.config.ts 
import { OrtoniReportConfig } from 'ortoni-report';

const reportConfig: OrtoniReportConfig = {
  logo: "path/logo.png",
  authorName: "LetCode Koushik",
  base64Image: false,
  preferredTheme: "dark",
  projectName: "Ortoni Demo Report",
  testType: "Smoke"
}
reporter: [['ortoni-report', reportConfig], ['dot']],

3.Ran the command to execute test code npx playwright test

Actual Result:- When test ran, i dont see the ortoni report generated.

Expected Result:- I should see the ortoni report
System details 
MAC SONOMA 14.5

<img width="690" alt="image" src="https://github.com/user-attachments/assets/f25175ec-6d5e-4009-8354-e2b671686bbd">
<img width="739" alt="image" src="https://github.com/user-attachments/assets/6767e593-3297-49f3-9739-d0c084a134f0">

I am sharing my code with you here
https://github.com/nirmalprojects1988/playwrightpracticeexamples/tree/otorniissue

I request you to review it and let me know

**Comment by ortoniKC** [comment]
I hope you have given the actual path of the logo?

**Comment by nirmalprojects1988** [comment]
Hello Kaushik,
Passing the logo path solved the problem and I can generate the report :) .

One suggestion may be that i am not the right person, but can we have a log for your future enhancement,where actually went wrong?
Before raising this issue I tried to remove the logo and author key , when I ran it ran without error and in the console I didn't see any log of error.
Saying that closing this bug
Thank you once again for your support.
